model/modeldecoder/modeldecoderutil/labels.go

```diff
@@ -44,8 +45,13 @@ func MergeLabels(eventLabels common.MapStr, to *model.APMEvent) {
 		to.Labels = make(model.Labels)
 	}
 	for k, v := range NormalizeLabelValues(eventLabels.Clone()) {
```

Comment:
Maybe get rid of this `NormalizeLabelValues(eventLabels.Clone())`, and do the transformation from `json.Number` in the switch? That will cut out some more allocations.